fandogh_cli/fandogh_client/__init__.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, response, message=None):
         if message:
             self.message = message
 
+        if self.response.json()['message']:
```

Comment:
raises error if:
- response is not json
- response is None
- response.json does not have attribute message